CMakeLists.txt

```diff
@@ -134,7 +134,7 @@ ENDIF()
 
 IF(WITH_VIGRANUMPY)
     set(WITH_BOOST "ON")
-    # set(WITH_BOOST_COMPONENTS ${WITH_BOOST_COMPONENTS} "python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
+    set(WITH_BOOST_COMPONENTS ${WITH_BOOST_COMPONENTS} "python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
```

Comment:
Could we just add branches for different platforms here? I am a little concerned that we get something we don't like by not pinning this somehow.